lib/Doctrine/Common/Collections/Criteria.php

```diff
@@ -76,6 +76,24 @@ public static function create()
     }
 
     /**
+     * Creates an instance with same behaviour as criteria parameters passed on ORM find methods.
+     *
+     * @param array $findCriteria
```

Comment:
Nope, this is the same as `ObjectRepository::findBy` method first parameter.

Ref: https://github.com/doctrine/common/blob/faf7d81273eeffb84d524006daf85d37a14ac42a/lib/Doctrine/Common/Persistence/ObjectRepository.php#L64

I'll add a comment.